wbooks/src/BookDetail.js

```diff
@@ -1,31 +1,60 @@
 import React, { Component } from 'react';
-import defaultBookIcon from './default_book.svg'
+import myBooks from './books.json';
+import { Link } from 'react-router-dom';
+import defaultBookIcon from './default_book.svg';
 import './BookDetail.css';
 
 class BookDetail extends Component {
-  constructor() {
-    super()
-  }
-
   render() {
+    var book = myBooks.find(book => {
+      return book.id == this.props.match.params.id;
```

Comment:
Use `===`

Reply:
This can be written as:
```js
const book = myBooks.find(book => book.id === this.props.match.params.id);
```

You can write one line arrow functions like that, without `{}`, also the return is implicit.